scripts/tests/relay-graceful-shutdown.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "./scripts/tests/lib.sh"
+
+# Arrange: Setup a relayed connection
+install_iptables_drop_rules
+client_curl_resource "172.20.0.100/get"
+
+# Act: Send SIGTERM
+sudo kill --signal SIGTERM "$(pgrep firezone-relay)"
```

Comment:
Should this run inside the relay container?

Reply:
I think it could but it doesn't need to.